mathesar_ui/src/pages/table/TablePage.svelte

```diff
@@ -13,6 +12,7 @@
   import TableView from '@mathesar/systems/table-view/TableView.svelte';
   import ActionsPane from '@mathesar/systems/table-view/actions-pane/ActionsPane.svelte';
   import { makeSimplePageTitle } from '@mathesar/pages/pageTitleUtils';
+  import LayoutWithHeader2 from '@mathesar/layouts/LayoutWithHeader2.svelte';
```

Comment:
This component is no longer present. Please merge changes from the latest master and resolve conflicts.

Reply:
d80e96b1

Reply:
d80e96b1